account_partner_reconcile/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,2 @@
+The button is visible only to users that belong to the accounting groups
+"Accountant" or "Adviser".
```

Comment:
"Billing" or "Billing Manager". To carry out the matching, users need to have activated the check "Show Full Accounting Features".